### ISSUE TITLE
[ep-matrix] Integration the Matrix room chat with StreamYard

### ISSUE THREAD

**Issue by malemburg** [EuroPython/epcon#1407]
This is not essential, but would be nice to have.

StreamYard has integrations for several chat systems, but not for Matrix.org

**Comment by gilgamezh** [comment]
Did you manage to find documentation on how to integrate?

**Comment by malemburg** [comment]
No. It seems we'd have to use one of the supported systems and replay chat entries there in order for StreamYard to pick them up. Not sure whether it's worth it.
